### PR TITLE
Removed unnecessary tests in release.yml

### LINKED ISSUE #2696: Make release faster by removing tests in github workflow
**What would you like to be added**:

When we release telescope, it also runs our eslint, prettier, unit and e2e tests which are redundant as we already check on opening PR and on merging to master. 

**Why would you like this to be added**:

Remove tests running in this file [.github/workflows/release.yml](https://github.com/Seneca-CDOT/telescope/blob/master/.github/workflows/release.yml)

### PR DESCRIPTION
**Pull request by TueeNguyen**
## Issue This PR Addresses

This fixes #2696 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

- Removed unnecessary tests in `release.yml` to make releasing telescope faster. 
- Edited `release.md` to reflect on this change
## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by tpmai22 — DISMISSED** [review]

**Comment by tpmai22** [comment]
Lgtm

**Review by JiaHua-Zou — DISMISSED** [review]

**Review by tpmai22 — APPROVED** [review]

**Review by DukeManh — DISMISSED** [review]
👍

**Review by humphd — CHANGES_REQUESTED** [review]
Let's update the release.md doc to indicate that no tests will be run.  This means that whoever does the release and makes a new tag, **must** check that current master is green, and that they are tagging the same commit sha as what was built on master.

There's a bit more personal responsibility to doing it this way, if we remove the tests.  We need to be clear about it.

File: docs/release.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ We use the [npm-version](https://docs.npmjs.com/cli/v6/commands/npm-version) com
 To create a new release, follow these steps:
 
 1. Make sure your `master` branch is up-to-date, and you have the most recent git tags in your repo: `git pull upstream master --tags`.
-1. Make sure your working tree is clean.
+1. Make sure your working tree passes all checks (indicated by the green marker) and is clean.
```
Comment (humphd) [inline docs/release.md:31]: `1. Make sure that the latest commit on `master` is passing CI on GitHub Actions, and that you are sitting on the same commit locally (i.e., `git show` or `git log` and compare the git SHA).`

**Review by humphd — CHANGES_REQUESTED** [review]

**Review by humphd — APPROVED** [review]
Looks good, needs a rebase.

**Review by cindyledev — APPROVED** [review]
Looks good. Thank you for doing this :)

**Review by humphd — APPROVED** [review]

**Review by DukeManh — APPROVED** [review]

**Comment by TueeNguyen** [comment]
Thanks guys!

**Pull request merged**